### ISSUE TITLE
Screen reader does not announce decimal values for Decay Time correctly

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/alpha-decay#48]
**Test device**
Asus Zenbook
**Operating System**
Win 11
**Browser**
Firefox
**Problem description**

From https://github.com/phetsims/qa/issues/1414
When the Decay Time includes a decimal value, the screen reader does not announce it correctly. For example, if the displayed value is 5.3, the screen reader announces 5.0 instead.

The screen reader should announce the actual decimal value shown in the interface so that the spoken feedback matches the displayed value.

**Steps to reproduce**
1. Go to the Single Atom screen.
2. Change to Custom Isotope mode
3. Click the Add Atom button until an atom with a decay time containing a non-zero decimal value appears (e.g., 5.3 or 2.1).

**Visuals**

https://github.com/user-attachments/assets/9ea7dbc5-aea7-4a00-bad6-ff7c8746816c

<details>
Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Alpha Decay‬
URL: https://phet-dev.colorado.edu/html/alpha-decay/1.0.0-dev.4/phet/alpha-decay_all_phet.html
Version: 1.0.0-dev.4 2026-07-09 00:01:34 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/150.0.0.0 Safari/537.36
Language: en-US
Window: 1600x812
Pixel Ratio: 1.7999999523162842/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}
<details/>

**Comment by AgustinVallejo** [comment]
Thanks! This helped me identify and solve another known issue. It should be good now. Assigning back

**Comment by vperezcadavid99** [comment]
Thanks @AgustinVallejo ! but...checked on main and the issue seems to keep (tested on Firefox + NVDA)

https://github.com/user-attachments/assets/b22ba0e0-8dd0-4edf-b342-f8b595bb24c2

**Comment by AgustinVallejo** [comment]
Hmm, either I cannot seem to replicate or this was fixed with the other decimal places issue. Can you check again?

**Comment by AgustinVallejo** [comment]
Nevermindd, I found it and hopefully solved it. Back to you Valen.

**Comment by vperezcadavid99** [comment]
Thanks @AgustinVallejo
Looks much better. Closing!
